src/lib/viewers/BaseViewer.js

```diff
@@ -344,7 +344,10 @@ class BaseViewer extends EventEmitter {
         // Add a resize handler for the window
         document.defaultView.addEventListener('resize', this.debouncedResizeHandler);
 
-        this.addListener('load', () => {
+        this.addListener('load', (event) => {
+            // this.scale set to 1 if event.scale does not exist
+            ({ scale: this.scale = 1 } = event);
```

Comment:
That would indeed make more sense and I wish it would work. Issue here is as follows:

BaseViewer gets scale from 'scale' event from BaseViewer child class
Annotator gets scale from data-scale which it also sets. Which creates a cold start problem. Annotator.init() creates the annotatedElement and sets data-scale to be 1 at the time of initialization. init() then renders the annotations. Afterwards, data-scale is correctly set through BaseViewer's scale listener doing Annotator.setScale(). At this point, annotations have been shown and they are not redrawn.